i18n/en.json

```diff
@@ -3186,43 +3186,7 @@
   "login_mfa.submit": "Submit",
   "login_mfa.submitting": "Submitting...",
   "login_mfa.token": "MFA Token",
-  "login.changed": " Sign-in method changed successfully",
-  "login.create": "Create one now.",
   "login.createTeam": "Create a team",
-  "login.email": "Email",
```

Comment:
I don't think these should be removed, something is wrong here.

Reply:
I don't know what to say to that. I will fix the other comments but about this, should I revert to my previous commit then start to fix it. But even then the i18n will fail. I tried to run the `make i18n-extract` and first I thought too about these things shouldn't be removed but then I figure maybe that's why its failing. Let me know what to do then.